### ISSUE TITLE
The DINO baseline is higher than STEGO.

### ISSUE THREAD

**Issue by xiao7199** [mhamilton723/STEGO#58]
Thanks for sharing the code for your exciting work. 

I have a question regarding the linear probing result of the DINO pipeline. 

Since there are no details in the paper for evaluating the baseline, I write my own training script for linear probing and plug the learned classifier into the evaluation script provided in this repo for a fair comparison.  From my experiment, DINO gets 42.73 mIOU, and STEGO gets 42.13 mIOU for COCO stuff, which means STEGO is not outperforming the DINO baseline for linear probing.   
Can you provide more details on reproducing the baseline result for linear probing? 
Thanks.

**Comment by mhamilton723** [comment]
Its been a while since i ran ts baseline but i think i set `projection_type` to None and dropout to 0 and then trained and evaluated the system as usual. No projection means that no segmentation head is applied so its just DINO.

Let me know if this repros the numbers for you. If you indeed found that the pro code could be improved that is interesting as well. Thanks for your patience!

**Comment by axkoenig** [comment]
Hi there, 
congrats on the great paper! We are publishing a [follow-up study on STEGO](https://arxiv.org/abs/2304.07314) in CVPR 23 Workshops, which also looks into the issue you describe. Table 2 might be interesting to you! :)
Cheers, Alex

**Comment by xiao7199** [comment]
Hi Alex, 

Thanks for sharing the draft and the results. I think the results from Table 2 are consistent with what I've found. 

Best, Xiao.
